.github/workflow-scripts/gh-pages/deploy.js

```diff
@@ -151,6 +175,34 @@ async function undeploy({ branch, repo, owner, github, exec }) {
         console.log(e);
       }
     }
+
+    const { data: { environments } } = github.rest.repos.getAllEnvironments({
```

Comment:
This block of code removes the environments created by the deployments: https://github.com/Legal-and-General/canopy/settings/environments